pkg/agent/datamodel/types.go

```diff
@@ -1298,6 +1298,7 @@ type NodeBootstrappingConfiguration struct {
 	HTTPProxyConfig                *HTTPProxyConfig
 	KubeletConfig                  map[string]string
 	EnableRuncShimV2               bool
+	GPUInstanceProfile             string
```

Comment:
for future reference, if you need to plumb a type into agentbaker, it's way easier to do the updates in agentbaker first, because they don't depend on anything from RP -- just make sure the logic is a no-op when the new field hasn't been passed from RP

that way, you can do:
- agentbaker api changes
- agentbaker logic changes (if it's a big feature, nice to separate)
- RP agentbaker vendor with no other changes (will be done for you in weekly vhd/vendor)
- RP api changes + type conversion to agentbaker
- additional RP logic, validation, new field values, etc etc